.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Julia Development",
+    "build": {
+        "dockerfile": "Dockerfile"
+    },
+    "features": {
+        "ghcr.io/julialang/devcontainer-features/julia:1": {}
+    },
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "julialang.language-julia"
+            ]
+        }
+    }
+    //"postCreateCommand": "echo 'Test command'"
+    //"postCreateCommand": {
+    //    "1-julia-activate": "/usr/local/julia-devcontainer-features/postcreate.jl",
+    //    "2-julia-packages": "julia --startup-file=no -e 'using Pkg; Pkg.add([\"PkgTemplates\", \"JuliaFormatter\"]); Pkg.precompile()'"
```

Comment:
https://github.com/daytonaio/daytona/pull/1393

Raised a PR in Daytona that will resolve the issue for reading the devcontainer when you have single quotes inside. I'll let you know once we release this today.

Nontheless, why isn't this line a part of the dockerfile like here https://github.com/daytonaio/sample-julia-genie/pull/1/files#diff-13bd9d7a30bf46656bc81f1ad5b908a627f9247be3f7d76df862b0578b534fc6R17?

If it was part of the dockerfile, it would get built into the image so the user would have the packages there immediately every time they create a workspace from that sample. It would improve the startup time of the workspace.

Reply:
My opinion is that we should avoid as much as possible to use version numbers of the Docker image (for maintenance purposes) like

    FROM julia:1.11.1-bullseye

in samples because it will force you to update these examples so often!

Try to put this line (adding PkgTemplates...) in the Dockerfile... you will see that it won't work here because julia won't be installed at this stage (because I'm using here devcontainer features... not directly Docker image)

Reply:
Ah okay, you're right. That's fine for this sample then.

I'll resolve the comment once we release the fix we mentioned and once you uncomment this then.

Reply:
@s-celles you can update daytona now and then update the provider with `daytona provider update`. The new version should have no issues with this repo now.